models/job.go

```diff
@@ -0,0 +1,54 @@
+package models
+
+// Job for getting job info.
+type Job struct {
```

Comment:
There is already a struct in clients/api_utility.go called JobResp. Looks like u have duplicated it.